src/navigation/index.ts

```diff
@@ -183,18 +181,13 @@ export const Stacks: { [key in Stack]: Stack } = {
 export type StatusBarStyle = "dark-content" | "light-content"
 
 export const useStatusBarEffect = (statusBarStyle: StatusBarStyle): void => {
-  const backgroundColor =
-    statusBarStyle === "dark-content"
-      ? Colors.primaryLightBackground
-      : Colors.primary125
-
   useFocusEffect(
     useCallback(() => {
       StatusBar.setBarStyle(statusBarStyle)
       if (Platform.OS === "android") {
-        StatusBar.setBackgroundColor(backgroundColor)
+        StatusBar.setBackgroundColor("transparent")
         StatusBar.setTranslucent(true)
```

Comment:
Is this redundant now? I remember @johnschoeman and I testing this a few weeks ago and it didn't seem to do anything.

Reply:
The good news is that it actually is doing something 🎉